dashboards/velero-prometheus/velero-prometheus.json

```diff
@@ -0,0 +1,520 @@
+{
+  "name": "Velero-Prometheus",
+  "description": null,
+  "permissions": "PRIVATE",
```

Comment:
Our validation points to this line in the dashboard file - Our services don't allow the `permissions` field to be in the `json`.

Removing this should give all green checkmarks for validation!

Reply:
thank you for catching that. I've updated and removed the account id array elements from the filters as well.